qp_shogun/shogun/tests/test_shogun.py

```diff
@@ -174,18 +177,36 @@ def test_generate_shogun_dflt_params(self):
                 'Capitalist': False,
                 'Number of threads': 15,
                 'Percent identity': 0.95},
-            'rep82_utree': {
-                'Database': join(self.db_path, 'rep82'),
-                'Aligner tool': 'utree',
+            # 'rep82_utree': {
```

Comment:
Can you remove these commented-out lines?

Reply:
I prefer to keep them as there is a chance that in a future release we will need to readd ...